Add new view `model.vw_model_run_type` to support junk model runs (#1071)
ccao-data/data-architecture: dbt/models/model/docs.md

**Primary Key**: `year`, `meta_pin`, `meta_card_num`
{% enddocs %}

# vw_model_run_type

{% docs view_vw_model_run_type %}
View combining metadata from both the production `model` database and the
development `z_dev_model` database.

> **Primary Key**: `year`, `run_id`

Author: This is just because model.metadata is keyed to these, but shouldn't it just be run_id?

Reviewer: **[Suggestion, optional]** Yeah, I believe that's right. No harm in correcting it for this table.

```suggestion
**Primary Key**: `run_id`
```

**Primary Key**: `year`, `meta_pin`
{% enddocs %}

# z_dev_model.metadata

{% docs table_z_dev_model_metadata %}

> Developmental counterpart to `model.metadata`. Used for junk runs that are not intended for stored indefinetely. Contents are deleted after 30 days.

Reviewer: **[Nitpick, optional]** A couple typos:

```suggestion
Development counterpart to `model.metadata`. Used for junk runs that are not intended to be stored indefinitely. Contents are deleted after 30 days.
```

> **Primary Key**: `year`, `run_id`

Reviewer: **[Suggestion, optional]** Same here as above:

```suggestion
**Primary Key**: `run_id`
```
